src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -110,14 +106,17 @@ class SparqlQleverVisitor {
   // meaning of blank and anonymous nodes is different.
   bool isInsideConstructTriples_ = false;
 
+  // Resets the Visitors state between updates. This resets everything except
+  // prefix and base.
```

Comment:
```suggestion
  // prefix and base, because those are shared between consecutive updates.
```

Reply:
And call this function `resetStateForMultipleUpdates` to make it clearer, that this function has an explicit purpose.